zoo/example.txt

```diff
@@ -0,0 +1,24 @@
+Basic usage:
+
+Set up of a simple zoo with two cages for testing.
+
+from zoo.zoo import Zoo, Cage, BaseSpecies, Lion, Hyena, Wildebeest, Gazelle
+zoo = Zoo('Test Zoo')
+zoo.__repr__  # Displays the name of the zoo alongside the current number of cages and animals
+one_cage = zoo.add_cage('One Cage')  # Cages can be created directly from a Zoo object passing only their name.
+another_cage = zoo.add_cage('Another Cage')
+
+one_cage.__repr__  # Displays the cage name alongside the animals that are currently in it.
+one_cage.add_animal(Lion('Lilly'))  # Animals can be created directly from a cage or independently.
+one_cage.add_animal(Hyena('Henny'))  # In this case the cage_relationships method checks whether animals are eaten. In this case there are none.
+one_cage.add_animal(Wildebeest('Willy'))  # In this case Willy is instantly eaten.
```

Comment:
You could simplify your API to begin with by just doing something like:

    willy = Wildebeest('Willy')
    one_cage.add_animal(willy)

Then you don't need the get_animal method on the zoo class.